client/apps/edit/client.coffee

```diff
@@ -24,6 +24,6 @@ HeroSection = React.createFactory require './components/hero_section/index.coffe
     $('#edit-sections')[0]
   )
   React.render(
-    HeroSection(section: article.heroSection)
+    HeroSection(section: article.heroSection, hasSection: article.get('section_ids')?.length > 0)
```

Comment:
This is the prop that gets passed in to determine if the fullscreen icon should be disabled or not